LPS-115862 Before submiting the form, check if the inputs should be cleaned. This is done when the language selected in the moment of submitting is not translated, otherwise a new translation is generated

victorg1991 opened the pull request.

Sharryxue committed:
LRQA-59529 Add macro to add announcement entry via json

Sharryxue committed:
LRQA-59529 Add macro to make it able to add announcement entry via json

alee8888 committed:
LPS-115779 Simplify AccountRoleLocalServiceImpl to use OrderFactoryUtil.addOrderByComparator

rebsilva committed:
LPS-113187 Setting a Max Width of 808px for the Form Container

rebsilva committed:
LPS-113187 Setting margins of 12px for fields

rebsilva committed:
LPS-113187 Centering the Form Container

rebsilva committed:
LPS-113187 Fixing margin-left for fields

ambrinchaudhary committed:
LPS-115921 Style the languages selector

dgarciasarai committed:
LPS-115593 Use Lexicon colors

dgarciasarai committed:
LPS-115593 Add axis names

ambrinchaudhary committed:
LPS-115921 Add language's icons

ambrinchaudhary committed:
LPS-115921 Add new messages

ambrinchaudhary committed:
LPS-115921 BuildLang

pavel-savinov committed:
LPS-116094 Use group ID from the vocabulary when exists

ambrinchaudhary committed:
LPS-115921 SF

4lejandrito committed:
LPS-115921 Auto SF

4lejandrito committed:
LPS-115921 Chaining not allowed here

4lejandrito committed:
LPS-115921 Using LocaleUtil function instead

4lejandrito committed:
LPS-115921 Inlining to make it more readable

4lejandrito committed:
LPS-115921 Not needed

victorg1991 committed:
LPS-115862 Before submiting the form, check if the inputs should be cleaned. This is done when the language selected in the moment of submitting is not translated, otherwise a new translation is generated

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

4lejandrito committed:
LPS-115921 Locales are objects, languageIds are the ids

4lejandrito committed:
LPS-115921 This must be lowercase

4lejandrito committed:
LPS-115921 Removing the source language from the target list

4lejandrito committed:
LPS-115921 Using constants

4lejandrito committed:
LPS-115921 SF

rebsilva committed:
LPS-113187 Moving changes to the app builder file

rebsilva committed:
LPS-113187 Fixing the margin between the fields

4lejandrito committed:
LPS-115921 SF

linolaoj committed:
LPS-115931 setting focus to select field after chose an option

ambrinchaudhary committed:
LPS-116148 Reduce the size of the language selectors

ambrinchaudhary committed:
LPS-116148 Reduce margins between buttons

ambrinchaudhary committed:
LPS-116148 Add direction to the localized inputs

victorg1991 committed:
LPS-115862 When publishing the default language count the input as translated

marcosapmf committed:
LPS-116172 define color icon and title

marcosapmf committed:
LPS-116172 enable list component to color type

marcosapmf committed:
LPS-116172 create color component

marcosapmf committed:
LPS-116172 use color component as list item if field type is color

samanzanedo committed:
LPS-116218 Method refactoring for getting canonical file

jonathanmccann committed:
LPS-116250 Do not update the password modified date otherwise the user will be logged out since their session is older than their password. Partially reverts the changes made by LPS-85179 since LPS-85179 is still reproducible even with those changes in place

pavel-savinov committed:
LPS-116094 Show asset vocabularies inherited from Global site

pavel-savinov committed:
LPS-116094 Don't allow edit inherited vocabularies/categories

ambrinchaudhary committed:
LPS-116197 Add styles for the mobile version

dgarciasarai committed:
LPS-115057 Add empty state when there are no categories selected

dgarciasarai committed:
LPS-115057 Build Lang

IstvanD committed:
LPS-116103 Extract ItemSelectedEventName from the shortened Item Selector URL

Commit:
LPS-116295 create dialog variable and centerDialog() to be able to re-center from different scope

Commit:
LPS-116295 add resize event listener to boundingWindow and debounce it

Commit:
LPS-116295 clean up on destroyPortlet

4lejandrito committed:
LPS-116231 Explicitly replacing

4lejandrito committed:
LPS-115921 Capitalizing From

See https://github.com/brianchandotcom/liferay-portal/pull/90673\#issuecomment-650339407

4lejandrito committed:
LPS-115921 buildLang

pavel-savinov committed:
LPS-116031 New methods to get info field sets based on the asset entry and scope group

pavel-savinov committed:
LPS-116031 Semantic versioning

pavel-savinov committed:
LPS-116031 Create categorization info field set with available vocabularies as info fields

pavel-savinov committed:
LPS-116031 Provide vocabulary-specific info field values for categorization field set

jkappler committed:
LPS-116094 groupId is only used when adding a category, so move it inside the if

pavel-savinov committed:
LPS-116031 Support new method for the web content

pavel-savinov committed:
LPS-116031 Support new method for blog entries

gamerson committed:
LPS-111461 fix excludes

IstvanD committed:
LPS-116103 Adjust test for shorter Item Selector URL

holatuwol committed:
LPS-116316 ServletContext.getRequestDispatcher requires that the path start with a slash

hhuijser committed:
LPS-112607 Move class to *.constants

hhuijser committed:
LPS-112607 Update usage

hhuijser committed:
LPS-115716 Move class to *.constants

hhuijser committed:
LPS-115716 Update usage

hhuijser committed:
LPS-115716 Move class to *.constants

hhuijser committed:
LPS-115716 Update usage

hhuijser committed:
LPS-115716 Move class to *.constants

hhuijser committed:
LPS-115716 Update usage

hhuijser committed:
LPS-115716 Move class to *.constants

hhuijser committed:
LPS-115716 Update usage

hhuijser committed:
LPS-115716 Use latest project-templates-extensions

Jeremy-Ch committed:
LRQA-59514 Update tests and macros to reduce the pause time

victorg1991 committed:
LPS-116156 Make possible to pass className to the topper

victorg1991 committed:
LPS-116156 Send a props to the topper children to check if they are wrapped in a topper

victorg1991 committed:
LPS-116156 If the container is within a topper add the margin classes to the topper

ambrinchaudhary committed:
LPS-116197 Apply changes in the import translation page

dgarciasarai committed:
LPS-115593 Better readability

dgarciasarai committed:
LPS-115593 Better positioning css

dgarciasarai committed:
LPS-115593 Refactor

marcellustavares committed:
LRAC-6161 SF

michaelhashimoto committed:
LRCI-1368 Add build to get the portal release for usage in spira

michaelhashimoto committed:
LRCI-1368 Add method to builds to get pull request objects

michaelhashimoto committed:
LRCI-1368 Add build to get the portal fixpack release for usage in spira

michaelhashimoto committed:
LRCI-1368 SF

Commit:
LPS-113750 Adjusted sidebar-body height

shuyangzhou committed:
LPS-116218 SF

thektan committed:
LPS-114556 portal-search-web: Fixes untranslated language key

Must specifically use `Liferay.Language.get` to properly filter and replace the language key.

chaowang-123 committed:
LRQA-59828 Update the locator

Sharryxue committed:
LRQA-59529 Update the tests use json to add entry

Sharryxue committed:
LRQA-59834 Open the product menu to initialize the site

4lejandrito committed:
LPS-116398 Running after portlets are initialized to ensure LayoutFriendlyURL model listeners are active

IstvanD committed:
LPS-116355 Adding language key

IstvanD committed:
LPS-116355 buildLang

IstvanD committed:
LPS-116355 Applying BlogsSubscriptionSender

jkappler committed:
LPS-116094 Correct naming

jkappler committed:
LPS-116094 Avoid calling the method all the time since we already have the object

IstvanD committed:
LPS-116355 Adding configuration option for sending notifications to the Blog Entry creator

IstvanD committed:
LPS-116355 baseline

marcelabc committed:
LPS-116144 Make showLabel property visible on the sidebar

marcelabc committed:
LPS-116144 Fix dataType property for numeric types

matuzalemsteles committed:
LPS-115267 Enables FormBuilder testing

Some tests were skipped because some are failing when they have activated again, some is because they changed the behavior of the components and did not update the tests.

carolmariaabb committed:
LPS-116172 Use ternary operator instead of the listItem

inacionery committed:
LPS-115014 Disable search button

rafaprax committed:
LPS-116439 | LPS-110981 Add tests

Commit:
LPS-116031 Do not try to guess that the invoker wants a scopeGroupId to be taken into account.

If they wanted that, they should use the new method and pass it explicitly. If no scopeGroupId or classTypeId is provided, let's just return the fields that are generic regardless of the scope and class type.

Commit:
LPS-116031 Rename method to add "Asset" as a prefix for consistency

Commit:
LPS-116031 Extract method getAssetVocabularies()

I think this is more clear since there is another method of the same name that receives an AssetEntry as a parameter. By making this code also a method it's more explicit that both return a different set of vocabularies based on different information provided.

Commit:
LPS-116031 Let's keep consistency with what the asset-categories-selector taglib generates which is a single fieldset and which is usually labelled "Categorization"

Commit:
LPS-116031 Change the label of the existing categories field to highlight the difference with the new fields which are per vocabulary. The pre-existing fields will return all categories regardless of the vocabulary.

Note that we should not change the name of the field, since it would break existing mappings.

Commit:
LPS-116031 SF

matuzalemsteles committed:
LPS-115267 Disable fragment shorthand temporary in some files

At the moment these files are transformed in FormBuilder tests as dependencies but we disable React presets to transform Metal + JSX files, we are temporarily using React.Fragment just so as not to cause transformation errors in FormBuilder tests.

brianchandotcom committed:
LPS-115921 As used

brianchandotcom committed:
LPS-115921 Inline whatever can be inlined

brianchandotcom committed:
LPS-115921 Sort

brianchandotcom committed:
LPS-115921 As used, better for the human brain

brianchandotcom committed:
LPS-115921 auto SF

interaminense committed:
LPS-113187 Set max-with for the app builder only

interaminense committed:
LPS-113187 Adjust form layout

Preston-Crary committed:
LPS-116468 Make JournalContentSearch and JournalFeed CT enabled.

Preston-Crary committed:
LPS-116468 Build Service.

Preston-Crary committed:
LPS-116468 Versioning.

marcosapmf committed:
LPS-115820 use JSONUtil to define DDMFormFieldValue values in tests

michaelhashimoto committed:
LRCI-1368 Fix unit tests

brianchandotcom committed:
artifact:ignore portal-impl 5.12.2 releng

brianchandotcom committed:
artifact:ignore portal-impl 5.12.2 prep next

brianchandotcom committed:
artifact:ignore portal-web 4.0.55 prep next

brianchandotcom committed:
artifact:ignore portal-web 4.0.55 releng

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore data-engine-lang 2.0.23 change log

brianchandotcom committed:
artifact:ignore data-engine-lang 2.0.23 prep next

brianchandotcom committed:
artifact:ignore data-engine-lang 2.0.23 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 5.0.30 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 5.0.30 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-field-type 5.0.30 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-lang 5.0.28 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-lang 5.0.28 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-lang 5.0.28 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-service 5.2.5 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-service 5.2.5 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-service 5.2.5 artifact properties

brianchandotcom committed:
artifact:ignore export-import-service 7.0.13 change log

brianchandotcom committed:
artifact:ignore export-import-service 7.0.13 prep next

brianchandotcom committed:
artifact:ignore export-import-service 7.0.13 artifact properties

brianchandotcom committed:
artifact:ignore frontend-taglib-clay 4.0.3 change log

brianchandotcom committed:
artifact:ignore frontend-taglib-clay 4.0.3 prep next

brianchandotcom committed:
artifact:ignore frontend-taglib-clay 4.0.3 artifact properties

brianchandotcom committed:
artifact:ignore journal-lang 5.0.21 change log

brianchandotcom committed:
artifact:ignore journal-lang 5.0.21 prep next

brianchandotcom committed:
artifact:ignore journal-lang 5.0.21 artifact properties

brianchandotcom committed:
artifact:ignore translation-web 1.0.5 change log

brianchandotcom committed:
artifact:ignore translation-web 1.0.5 prep next

brianchandotcom committed:
artifact:ignore translation-web 1.0.5 artifact properties

brianchandotcom committed:
artifact:ignore portal-configuration-persistence-impl 3.0.10 change log

brianchandotcom committed:
artifact:ignore portal-configuration-persistence-impl 3.0.10 artifact properties

brianchandotcom committed:
artifact:ignore portal-configuration-persistence-impl 3.0.10 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore asset-taglib 5.0.21 change log

brianchandotcom committed:
artifact:ignore asset-taglib 5.0.21 prep next

brianchandotcom committed:
artifact:ignore asset-taglib 5.0.21 artifact properties

brianchandotcom committed:
artifact:ignore content-dashboard-web 1.0.9 change log

brianchandotcom committed:
artifact:ignore content-dashboard-web 1.0.9 prep next

brianchandotcom committed:
artifact:ignore content-dashboard-web 1.0.9 artifact properties

brianchandotcom committed:
artifact:ignore data-engine-taglib 2.5.5 change log

brianchandotcom committed:
artifact:ignore data-engine-taglib 2.5.5 artifact properties

brianchandotcom committed:
artifact:ignore data-engine-taglib 2.5.5 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-report-web 1.0.9 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-report-web 1.0.9 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-report-web 1.0.9 artifact properties

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 5.0.17 change log

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 5.0.17 artifact properties

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 5.0.17 prep next

brianchandotcom committed:
artifact:ignore redirect-web 1.0.18 change log

brianchandotcom committed:
artifact:ignore redirect-web 1.0.18 prep next

brianchandotcom committed:
artifact:ignore redirect-web 1.0.18 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore app-builder-web 1.0.40 change log

brianchandotcom committed:
artifact:ignore app-builder-web 1.0.40 artifact properties

brianchandotcom committed:
artifact:ignore app-builder-web 1.0.40 prep next

brianchandotcom committed:
artifact:ignore journal-web 4.0.33 change log

brianchandotcom committed:
artifact:ignore journal-web 4.0.33 artifact properties

brianchandotcom committed:
artifact:ignore journal-web 4.0.33 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 4.0.31 change log

brianchandotcom committed:
artifact:ignore layout-admin-web 4.0.31 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 4.0.31 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore app-builder-workflow-web 1.0.9 change log

brianchandotcom committed:
artifact:ignore app-builder-workflow-web 1.0.9 artifact properties

brianchandotcom committed:
artifact:ignore app-builder-workflow-web 1.0.9 prep next

brianchandotcom committed:
build.gradle auto SF

michaelhashimoto committed:
LRCI-1368 Use origin name for cache branch

javiergamarra committed:
LPS-113663 remove test

brookedalton committed:
LRQA-59414 Remove workaround

Commit:
LPS-116250 Only set passwordModifiedDate if the password actually changed

michaelhashimoto committed:
LRCI-1368 Avoid using BranchInformation to find the pull request url

Commit:
LPS-114698 Inline resource into application

CAustin582 committed:
LRIT-20631 Point to the public group for artifacts instead of individual repositories

BryanEngler committed:
LPS-116491 portal-search-elasticsearch7-impl: Add missing language key

https://issues.liferay.com/browse/LPS-116491

BryanEngler committed:
LPS-116491 portal-search-elasticsearch7-impl: buildlang

https://issues.liferay.com/browse/LPS-116491

Commit:
LRQA-59772 Revert "LRQA-59772 - Quarantine tests"

This reverts commit d1d54274cec032f24aed51718766903a65d052e9.

liuxingye committed:
LRQA-59897 Remove unnecessary properties

dellawang committed:
LRQA-59855 Update test to get site name

Commit:
LRQA-59905 Add locator for management bar in page editor

Commit:
LRQA-59905 Use new locator to make them robust

Commit:
LRQA-59906 Add if condition to make it robust

Commit:
LRQA-59909 Modify macro to correct parameter

brianchandotcom committed:
LPS-116094 SF

brianchandotcom committed:
LPS-116094 Sort

brianchandotcom committed:
LPS-116094 match service.xml for AssetCategory

brianchandotcom committed:
LPS-116094 normal order

pavel-savinov committed:
LPS-116031 Adjust tests, we have more fields now

victorg1991 committed:
LPS-116160 Add tooltip for preview button

dgarciasarai committed:
LPS-113904 Change icon color

IstvanD committed:
LPS-116355 Adding BlogsSubscriptionSender

sandrodw3 committed:
LPS-116156 Adapt xpath

jkappler committed:
LPS-116511 Don't use default user, but an admin group user to avoid permission problems when adding a portlet to a layout

jkappler committed:
LPS-116511 Instead of using existing portlets (which could be removed in the future), register new portlets for the specific test class and use them

4lejandrito committed:
LPS-116355 Auto SF

4lejandrito committed:
LPS-116355 Fixing SF complaints

4lejandrito committed:
LPS-116355 Simplifying

This is now specific to blogs so theres no need to pass the resource
name as a parameter.

Commit:
LPS-116295 get dialog from the event object and avoid creating an instance in centerDialog

Commit:
LPS-116295 declare the eventHandler and it's cleanup outside of AUI.use block

4lejandrito committed:
LPS-116355 Allowing configuration for notifications for current user in the SubscriptionSender

This is safer since existing logic doesn't have to be duplicated in
subclasses.

Commit:
LPS-116295 SF

4lejandrito committed:
LPS-116355 Using the original subscription sender with the new configuration set

adolfopa committed:
LPS-116103 Keep backwards compatibility and simplify regexp

adolfopa committed:
LPS-116103 SF

4lejandrito committed:
LPS-116355 Capitalizing message since it is a title

4lejandrito committed:
LPS-116355 buildLang

liulukiwi committed:
LPS-115976 Modify macros to delete fragment and go to configuration

4lejandrito committed:
LPS-116355 Sorting

4lejandrito committed:
LPS-116281 Sending the unmapped value

Sending the empty string causes the backend to fall back to the default
value.

marcosapmf committed:
LPS-116028 set bar chart labels font size

liferay-translation committed:
LPS-77699 Update Translations

liulukiwi committed:
LPS-115976 Add case to undo and redo page changes

victorg1991 committed:
LPS-113750 Calculate the value using constants instead of the hardcoded value

liulukiwi committed:
LPS-115976 Add situation for removing fragment from sidebar

liulukiwi committed:
LPS-115976 Add case to undo and redo master page changes in staging

inacionery committed:
LPS-116324 Remove trim for script

Commit:
LRQA-59951 | master

gabrielwas committed:
LPS-116456 Remove row just if it's a full empty row

petershin committed:
LPS-116492 Skip .lfrbuild-releng-ignore

brianchandotcom committed:
LPS-115820 Simplify, no need to be so fancy - Hugo SF, always call toString instead of toJSONString (impl is the same thing)

brianchandotcom committed:
LPS-115716 prep next

brianchandotcom committed:
LRCI-1368 prep next

wincent committed:
LPS-116520 Update alloy-editor to v2.11.9

Release notes since last update:

- https://github.com/liferay/alloy-editor/releases/tag/v2.11.9
- https://github.com/liferay/alloy-editor/releases/tag/v2.11.8

Most noteworthy fix:

- https://github.com/liferay/alloy-editor/pull/1404 (fixes toolbar
  visibility for very large tables).

brianchandotcom committed:
LPS-116031 pavel, please apply else where

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/etl/mulesoft/.gitrepo'.

CAustin582 committed:
LRIT-20631 Groups use a different URL pattern than repositories
